src/app/beer_garden/api/http/handlers/v1/garden.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+from brewtils.schema_parser import SchemaParser
+
+from beer_garden.api.http.authorization import authenticated, Permissions
+from beer_garden.api.http.base_handler import BaseHandler
+
+
+class GardenAPI(BaseHandler):
+    @authenticated(permissions=[Permissions.SYSTEM_READ])
+    async def get(self, garden_name):
+        """
+        ---
+        summary: Delete a specific Instance
```

Comment:
Need a new summary :smile:

Reply:
What?!? That one wasn't good enough... Fine it has been changed.